### ISSUE TITLE
Fails to install on CircleCI

### ISSUE THREAD

**Issue by martmists** [regebro/pyroma#14]
```
$ #!/bin/bash -eo pipefail
  python3 -m pip install snekchek

Collecting snekchek
  Downloading snekchek-0.1.3-py3-none-any.whl
...
Collecting pyroma (from snekchek)
  Downloading pyroma-2.3.tar.gz (345kB)
    100% |################################| 348kB 3.7MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-hgav_2ar/pyroma/setup.py", line 8, in <module>
        long_description=open("README.rst", 'rt').read() + "\n" +
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 3516: ordinal not in range(128)```

**Comment by martmists** [comment]
Possibly related: https://github.com/Kozea/cairocffi/pull/21

**Comment by martmists** [comment]
Seems to have been fixed in the latest commit, when does 2.4 release?
